nncf/common/graph/graph.py

```diff
@@ -252,6 +252,13 @@ def get_all_nodes(self) -> List[NNCFNode]:
             all_nodes.append(nncf_node)
         return all_nodes
 
+    def get_all_simple_paths(self, start_node_name: NNCFNodeName, end_node_name: NNCFNodeName):
```

Comment:
@asenina @vshampor introduced a new function to NNCFNetwork that is helpful for NAS in 2 places
without need to access a private `_nx_graph` field